examples/onchain-client/.env

```
@@ -0,0 +1,12 @@
+# This .env file is for demonstration purposes only.
+#
+# This should usually be excluded via .gitignore and the env vars attached to
+# your deployment enviroment, but we're including this here for ease of local
+# development. Please do not commit changes to this file!
+#
+# A different private key from the one that is used to deploy the world
+PRIVATE_KEY=0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6
+
+# Assuming this is the first World deployed by the `pnpm dev`.
+# Otherwise you need to change this.
+WORLD_ADDR=0x5FbDB2315678afecb367f032d93F642f64180aa3
```

as a general rule, let's always try to avoid abbreviations
```suggestion
WORLD_ADDRESS=0x5FbDB2315678afecb367f032d93F642f64180aa3
```